scripts/add_domains_and_exon_information_to_ensembl_transcript.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+import sys
+from collections import defaultdict
+
+if __name__ == "__main__":
+    # Read in transcript information file
+    transcripts = pd.read_csv("../data/ensembl_biomart_transcripts.txt", sep='\t')
+
+    ## Domains
+    domains_file = "../data/ensembl_biomart_pfam_grch37.p13.txt"
+    # Make a dictionary where every value added will be an empty list
+    # With this dictionary no keys have to be added for transcripts without domains
+    # This will automatically be empty list when transcript id (key in dict) is requested
+    domain_dict = defaultdict(list)
+
+    # Loop through each line of the file with pfam domain info and append information
+    # to dictionary, which has transcript id as key
+    with open(domains_file, 'r') as domains:
+        for line in domains:
+            # Split line and remove enter from last value
+            list_line = line.split("\t")
+            list_line[6] = list_line[6].replace("\n", "")
```

Comment:
Please use `strip('\n')` to remove the `\n`, for example:
`            list_line = line.split("\t").strip('\n')`